styles/fonts.css

```diff
@@ -16,6 +16,13 @@
   unicode-range: U+0000-00FF, U+0131, U+0152-0153, U+02BB-02BC, U+02C6, U+02DA, U+02DC, U+2000-206F, U+2074, U+20AC, U+2122, U+2191, U+2193, U+2212, U+2215, U+FEFF, U+FFFD;
 }
 
+@font-face {
```

Comment:
Please move  fallbacks to `styles.css`

Reply:
Since we are using `Poppins Fallback` for fallback font and that is already part of style.css. 
IMO we should keep this in fonts.css only and load it lazy,